scripts/setup_CMS_GEM.sh

```diff
@@ -44,7 +44,7 @@ if [[ !  -d $DIR_PIP ]]; then
 	#echo "-----"
 
 	#Installing packages
-	#pip2.7 install --user numpy scipy matplotlib ipython jupyter pandas sympy nose root_numpy
+	#pip2.7 install --user numpy scipy matplotlib ipython jupyter pandas sympy nose root_numpy xlrd
 	$DIR_PIP/bin/pip install --user numpy scipy matplotlib ipython jupyter pandas sympy nose root_numpy
```

Comment:
I think you want to add that to L48 instead since L47 is commented out.  While we are at it let's remove all commented code from this script.